Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/FocusHelper.js

```diff
@@ -0,0 +1,25 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+(function() {
+ "use strict";
```

Comment:
One *slight* nit is that the indentation seems to be messed up in this file (which would've happened if you tried editing in Xcode, hah). Since it is understood that the entire file is wrapped in a closure, that first-level of indentation can just be eliminated.